package.json

```diff
@@ -120,8 +120,8 @@
     "@storybook/manager-webpack5": "6.5.15",
     "@storybook/react": "6.5.15",
     "@storybook/theming": "6.5.15",
-    "@swc/cli": "0.1.62",
-    "@swc/core": "1.3.87",
+    "@swc/cli": "0.1.65",
+    "@swc/core": "1.4.7",
```

Comment:
Does this work require an SWC bump? or are you just killing two birds with one stone?

Reply:
good question. 

**context:**

I wanted to bump to latest in order to make sure `resolveFully` flag works as expected (its bit hard to lookup what changes were added when - folks dont have the best "changelog" experience unfortunately ).

**answer:**

seem it's not necessary to bump utilities 

<img width="420" alt="image" src="https://github.com/microsoft/fluentui/assets/1223799/16814a09-283f-4bae-a5b0-df3478157e01">

Reply:
It might not be optional, My project fails to compile on a clean install of the project as it can't import the SWC helper for export star. Please see #30795 for more details and a repro.